pwndbg/checksec.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+ Wrapper to checksec utility.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import sys
+import subprocess
+
+import pwndbg.file
+
+def checksec():
+    local_path = pwndbg.file.get_file(pwndbg.proc.exe)
+    result = {}
+
+    for program in ['checksec', 'checksec.sh']:
```

Comment:
Imho it is hard to read code with tons of ifs, so would be nice to reduce them if possible.

(also it is hard to read code which has a big if block and then else with one line, it is better to do the reverse - `if not program: error` and then continue the code without new indent/code block)